src/editors/containers/VideoEditor/components/VideoSettingsModal/components/CollapsibleFormWidget.jsx

```diff
@@ -3,6 +3,13 @@ import PropTypes from 'prop-types';
 
 import { Collapsible } from '@edx/paragon';
 
+/**
+ * Simple Wrapper for Form Widget components in the Video Settings modal
```

Comment:
```suggestion
 * Simple Wrapper for a Form Widget component in the Video Settings modal
```
nit: just was confused because I thought ALL the widgets went in this container based on the original phrasing, and was confused when I continued reading.